crypto/dkg.go

```diff
@@ -32,9 +32,12 @@ type DKGState interface {
 	Threshold() int
 	// Start starts running a DKG in the current node
 	Start(seed []byte) error
-	// HandleMsg processes a new message received by the current node.
+	// HandleBroadcastedMsg processes a new broadcasted message received by the current node.
 	// orig is the message origin index
-	HandleMsg(orig int, msg []byte) error
+	HandleBroadcastedMsg(orig int, msg []byte) error
```

Comment:
```suggestion
	HandleBroadcastMsg(orig int, msg []byte) error
```
A bit shorter and easier to pronounce